src/main/java/racingcar/view/ResultView.java

```diff
@@ -17,7 +19,7 @@ public static void printOutputStatement() {
   }
 
   public static void printOutput(Cars cars) {
-    cars.getValues().forEach(car -> System.out.println(car.toString()));
+    cars.getValues().forEach(car -> System.out.println(createOutput(car)));
     System.out.println();
   }
```

Comment:
차의 위치를 보여주는 메소드를 만드는건 어떠신가요?
```suggestion
  public static void printOutput(Cars cars) {
    cars.getValues()
            .forEach(ResultView::printCarPosition);
    System.out.println();
  }

  private static void printCarPosition(Car car) {
    String position = String.format(
            OUTPUT_FORMAT,
            car.getName(),
            MOVING_SIGN.repeat(car.getPosition())
    );
    System.out.println(position);
  }
```